### ISSUE TITLE
ion-datetime does not correctly render selected date

### ISSUE THREAD

**Issue by JStumpp** [ionic-team/ionic#14234]
**Ionic version:**  (check one with "x")
(For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[ ] **3.x**
[x] **4.x**
```
Problem in: "@ionic/core": "0.1.5-2"
Was not a problem in: "@ionic/core": "0.1.4-7"
```
**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request

**Current behavior:**

If the first list item is selected (either month, day or year) this item is not rendered or written to the value.

**Expected behavior:**

Showing the selected date.

**Steps to reproduce:**

Select the first item e.g. Januar, 1, 2017
Close with "Done"
Nothing is rendered. Try selecting e.g. Januar, 2, 2016 ==> The "2, 2016" is displayed, but not the Januar.

**Related code:**

```
<ion-item>
  <ion-label>Date</ion-label>
  <ion-datetime></ion-datetime>
</ion-item>
```

**Comment by FdezRomero** [comment]
I can also reproduce this issue with `@ionic/angular@4.0.0-alpha.7` and the new docs (locally).
Related to #14563.

Thanks,
Rodrigo

**Comment by pwespi** [comment]
For a possible solution, see https://github.com/ionic-team/ionic/pull/14479 and https://github.com/ionic-team/ionic/pull/14480.

**Comment by brandyscarney** [comment]
Duplicate of #14563 and fixed by d19061c863f8558a0e597e3a5a53f1567b271b7b. Will be in the beta.3 release, thanks for the issue!
